dist/index.js

```diff
@@ -1705,6 +1705,54 @@
       alpine$2(callback);
     };
 
+    var AlpineRangeMagicMethod = {
+      start: function start() {
+        checkForAlpine();
+        Alpine.addMagicProperty('range', function () {
+          return function (start, stop, step) {
+            if (step === void 0) {
+              step = 1;
+            }
+
+            // Accept $range(10) and expect 1...10
+            if (typeof stop === 'undefined') {
+              stop = start;
+              start = start ? 1 : 0;
+            } // Accept $range('a', 'f') and expect 97...102 (not 'a'...'f')
```

Comment:
Do we want to support this. It's a really peculiar use case and it doesn't translate back to char at the end.

Reply:
In PHP `range(1,5)` will output [1,2,3,4,5] so I used that. 

For the char stuff, I added it because it was mentioned on the [MDN example](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#Sequence_generator_range). Also, PHP implements it too (although it's converted back to chars). I felt it was more flexible not converting it back. 

> Do we want to support this?

I don't feel too strongly whether we do or not. I left it undocumented because I wasn't sure if it would be something requested or not.

I'm fine to remove it. What do you think?

I'll update the readme with some more examples. I'll add a demo to CodePen as soon as we make a release too.

Reply:
I think either we convert it back or we take it out for now. $range('a', 'c') printing "97,98,99" is a bit counterintuitive to me.
I would say we remove it for now until there's a real use case and people needing it, at that point we can assess the best path.

Reply:
I think either we convert it back or we take it out for now. $range('a', 'c') printing "97,98,99" is a bit counterintuitive to me.
I would say we remove it for now until there's a real use case and people needing it, at that point we can assess the best path.

Reply:
Makes sense to me.  Removed it.